resources/setups/qualification/device_module_config.json

```diff
@@ -30,6 +30,9 @@
     "ipaddr": {
       "timeout_sec": 120
     },
+    "dhcp": {
```

Comment:
Why are all these edits necessary?  We shouldn't have to make this change to so many files to make this happen -- so something is broken by either the process or the change.  It might be that we need to fix this before doing this PR because this is getting insane, but... we need a clear articulation of the problem first.  What happens if you don't make ll these "dhcp enabled" changes?